__tests__/buildTranslationFiles.spec.ts

```diff
@@ -30,17 +36,14 @@ function generateKeys({
   return keys;
 }
 
-function gConfig(
-  type: TranslationCategory,
-  config: Partial<Config> = {}
-): Config {
+function gConfig(type: TranslationCategory, config: Partial<Config> = {}) {
   return {
     input: [`${type}`],
     output: `${type}/i18n`,
     langs: ['en', 'es', 'it'],
     defaultValue: defaultValue,
     ...config,
-  };
+  } as Config;
```

Comment:
Why did we need to change to casting instead of declaring the return type?

Reply:
Well, I went back to your branch/commit.
But since we've set typescript to strict, if we don't do it that way, since the config passed as an argument is `Partial<Config>`, it's normal for the compiler to detect a potential error.

You'd have to add all the missing keys to the configuration to get it right (but we don't use them for testing).

Reply:
It's ok as is, I just forgot why. tnx